libcontainer/factory_linux_test.go

```diff
@@ -122,15 +122,18 @@ func TestFactoryNewTmpfs(t *testing.T) {
 	if m.Source != "tmpfs" {
 		t.Fatalf("Source of root: %s, expected %s", m.Source, "tmpfs")
 	}
-	unix.Unmount(root, unix.MNT_DETACH)
+	err = unix.Unmount(root, unix.MNT_DETACH)
+	if err != nil {
+		t.Error("failed to unmount root", err)
+	}
 }
```

Comment:
Could possibly ignore this one, unless we're concerned about things not being unmounted